Move API Generator into `@comet/api-generator` (#3445)
vivid-planet/comet: .changeset/tame-ligers-enjoy.md

---

> "@comet/api-generator": major

Reviewer: ```suggestion
"@comet/api-generator": major
"@comet/cms-api": major
```

---

> Api generator is now part of `@comet/cli` package and can be run with `comet api-generator` command.

Reviewer: We should have an upgrade script for this.

Author: i know, already on it.

Author: https://github.com/vivid-planet/comet-upgrade/pull/65

> Api generator is now in separated `@comet/api-generator` package and can be run with the same command as before `comet-api-generator` command.

Reviewer: ```suggestion
Move API Generator into separate `@comet/api-generator` package

It can be run with the same `comet-api-generator` command as before.
```
